meta/main.yml

```diff
@@ -1,16 +1,18 @@
 ---
 
 allow_duplicates: yes
-dependencies: [ ]
+dependencies:
+  - sansible.nginx
```

Comment:
I don't think this should be here, we only want Nginx when running the tests, I would suggest having something like a test Galaxy requirements file that we add use via Molecule (or perhaps a test deps target in Makefile), the we add invoke the role in molecule/default/playbook.yml so it affects the tests only.

Reply:
I will revert to just creating the directories then.  I'm just testing whether the packages have been installed and the configuration files have been created anyway.